projects/client/RabbitMQ.Client/src/client/api/IConnectionFactory.cs

```diff
@@ -129,13 +129,6 @@ public interface IConnectionFactory
         IConnection CreateConnection(IList<string> hostnames, String clientProvidedName);
 
         /// <summary>
-        /// Advanced option.
-        ///
-        /// What task scheduler should consumer dispatcher use.
-        /// </summary>
-        TaskScheduler TaskScheduler { get; set; }
```

Comment:
This won't be an API change, but will break the behavior. If, for some reason, somebody used a `TaskScheduler` that limits the number of executed tasks it will no longer behave in the same manner.